### ISSUE TITLE
How to remove empty line in html attributes

### ISSUE THREAD

**Issue by liasica** [vuejs/eslint-plugin-vue#2570]
I wan't remove the empty line between `<BTable` and `  :columns="columns"`, what can I do?

Thanks :)

```html
<!-- Good -->
<BTable
  :columns="columns"
  :data="data"
  :scroll="scroll"
  :expandable="expandable"
/>

<!-- Bad -->
<BTable

  :columns="columns"
  :data="data"
  :scroll="scroll"
  :expandable="expandable"
/>
```

**Comment by FloEdelmann** [comment]
I don't think this is possible with eslint-plugin-vue yet. You can use Prettier or another code formatter.

For eslint-plugin-vue, which rule would you like to change? (see the list of all rules here: https://eslint.vuejs.org/rules/)
Or do you want to propose a new rule?
